### ISSUE TITLE
plugin: `add_control_to_dock` animated icons are not correct working in tree

### ISSUE THREAD

**Issue by MikeSchulze** [godotengine/godot#69330]
### Godot version

v4.0.beta6.official [7f8ecffa5]

### System information

Windows 11, Mac

### Issue description

I use animated textures for my TreeItems and they are not drawn correctly.
This problem only occurs when I add a scene to the plugin via `add_control_to_dock`, when I run the scene manually it works as expected.
The animation somehow gets stuck and only continues to run when I move the mouse over it or resize the window.
It looks like the control is not updating correctly in every frame.

![image](https://user-images.githubusercontent.com/347037/204478898-0d544651-ea9d-4f32-930a-2d3212806964.png)

### Steps to reproduce

load the attached example project

- run the plugin and look to the injected tree panel (animation is stuck)
- run the scene manually `res://TreePanel.tscn` it works 

### Minimal reproduction project

[animated_tree.zip](https://github.com/godotengine/godot/files/10111131/animated_tree.zip)

**Comment by TheDuriel** [comment]
The Editor by default will not redraw unless required. This is expected.

You can disable this in the EditorSettings, but it is enabled by default since it's considered a waste of your PC resources.

**Comment by kleonc** [comment]
As TheDuriel pointed out, that's expected. The relevant editor setting is: `interface/editor/update_continuously`.

However, instead of making the whole editor update continously I'd suggest to just call `queue_redraw()` when needed for whatever needs redrawing within the plugin. E.g. in the MRP something like this could be done in the `TreePanel.gd`:
```gdscript
func _process(_delta):
	if is_visible_in_tree():
		queue_redraw()
```
Nevertheless, I don't think there's anything to be done in here and this issue can be closed?

**Comment by MikeSchulze** [comment]
Thanks @kleonc i was thinking about to add a refresh onto the _progress, but is was feeling wrong.

**Comment by KoBeWi** [comment]
If it's AnimatedTexture, shouldn't it request redraw automatically?

**Comment by MikeSchulze** [comment]
> If it's AnimatedTexture, shouldn't it request redraw automatically? Does the same happen outside plugin?

yes is a AnimatedTexture, the scene itself runs without problems, it only occurs in the plugin added control

**Comment by KoBeWi** [comment]
I think without "update continuously", the editor should still redraw when requested (this actually happens when the node is inside a scene). Not sure why it wouldn't work here.

> The Editor by default will not redraw unless required. 

Well, it is required in this case. If I put an animated node in editor UI, I'd expect it to still animate. The "redraw only when required" refers only to static stuff (and all default editor's controls are static).

**Comment by MikeSchulze** [comment]
> Well, it is required in this case. If I put an animated node in editor UI, I'd expect it to still animate. The "redraw only when required" refers only to static stuff (and all default editor's controls are static).

this was also my assumption, which is why I had created this ticket ;)

**Comment by kleonc** [comment]
BTW if `interface/editor/show_update_spinner` editor setting is enabled and trying to enable "Update Continuously" by clicking on that spinner and choosing such option from the popup menu:
![godot windows editor dev x86_64_ih9yqjBsbu](https://user-images.githubusercontent.com/9283098/204579394-bab738cf-1549-4603-b85b-8935690cc1e9.png)
it actually shows this popup:
![godot windows editor dev x86_64_0NL54uR8uV](https://user-images.githubusercontent.com/9283098/204579995-a9ddaab4-b6cb-4c0e-8384-d6321465bac7.png)

But nothing like that is shown when enabling "Update Continuously" through the editor settings.

**Comment by MikeSchulze** [comment]
this bug is still exists ;(

**Comment by KoBeWi** [comment]
It's the same issue as #77415
AnimatedTexture is currently deprecated and this won't change at least until https://github.com/godotengine/godot-proposals/issues/7379 is implemented.

**Comment by MikeSchulze** [comment]
Ok thanks for the update
